lookout/style/format/analyzer.py

```diff
@@ -256,6 +256,7 @@ def generate_fixes(self, data_service: DataService, changes: Sequence) -> Iterab
                         find_deleted_lines(prev_file, file),
                     )))
                 fe = FeatureExtractor(language=lang, **rules.origin_config["feature_extractor"])
+                fe.cutoff_label_support = 0  # ad-hoc for https://github.com/src-d/style-analyzer/issues/387  # noqa: E501
```

Comment:
sorry for the nitpick but here we go: we should put comment line above to avoid noqa